.github/workflows/ci.yml

```diff
@@ -9,8 +9,6 @@ name: CI
 
 on:
   push:
-    branches:
-      - master
```

Comment:
Nit: I actually like to only run the build for PRs and pushes to the main dev branch. It's a poor man's solution to "duplicated" builds ;-) (more: https://github.com/mockito/mockito/pull/2103)